### PR TITLE
Improve coverage to search package method

### PR DESCRIPTION
**Pull request by adeliaferreira**
# Description
Add a new test case to cover error on web service on search package function

FIXES: THEEDGE-3058

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [X] Tests update
- [ ] Refactor

### CODE REVIEW

File: pkg/clients/imagebuilder/client_test.go [hunk 1]
```diff
@@ -85,6 +85,18 @@ var _ = Describe("Image Builder Client Test", func() {
 		Expect(err).To(BeNil())
 		Expect(res.Meta.Count).To(Equal(0))
 	})
+	It("test web service error while search a package", func() {
```
Comment (mgold1234) [inline pkg/clients/imagebuilder/client_test.go:88]: maybe better to write 'while there is no package to search" WDYT?

Comment (adeliaferreira) [reply pkg/clients/imagebuilder/client_test.go:88]: This new test case will test the http fail status ( Bad request) from web service. Maybe "test web service failure on search package" could be a better string?

Comment (AaronH88) [reply pkg/clients/imagebuilder/client_test.go:88]: "StatusBadRequest" seems to be the most important part of this test, and in my opinion, the test name should reflect this.
Consider something like "test web service when package search returns StatusBadRequest"

Comment (adeliaferreira) [reply pkg/clients/imagebuilder/client_test.go:88]: Sounds good to me the @AaronH88 suggestion, are you good with this nome @mgold1234 ?

Comment (adeliaferreira) [reply pkg/clients/imagebuilder/client_test.go]: It's already update, if you can recheck, it'll be amazing. Thanks everyone

**Review by AaronH88 — APPROVED** [review]
/lgtm

File: pkg/clients/imagebuilder/client_test.go [hunk 2]
```diff
@@ -85,6 +85,18 @@ var _ = Describe("Image Builder Client Test", func() {
...
 		Expect(res.Meta.Count).To(Equal(0))
 	})
+	It("test web service when package search returns StatusBadRequest", func() {
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "application/json")
+			w.WriteHeader(http.StatusBadRequest)
+			fmt.Fprintln(w, `{"error":{"count":0}}`)
+		}))
+		defer ts.Close()
+		config.Get().ImageBuilderConfig.URL = ts.URL
```
Comment (ldjebran) [inline pkg/clients/imagebuilder/client_test.go:95]: we need to restore the initial config imagebuilder url

Comment (adeliaferreira) [reply pkg/clients/imagebuilder/client_test.go:95]: sorry, don't understood, can you explain better?

Comment (adeliaferreira) [reply pkg/clients/imagebuilder/client_test.go:101]: Added, thanks for your suggestion

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Review by ldjebran — APPROVED** [review]
lgtm

**Pull request merged**